src/doc/book/conditional-compilation.md

```diff
@@ -41,8 +41,12 @@ they get set in the [`[features]` section][features] of your `Cargo.toml`:
 # no features by default
 default = []
 
+# Add feature "foo" here, then you can use it. 
+# Our "foo" feature depends on nothings else.
+foo = []
+
 # The “secure-password” feature depends on the bcrypt package.
-secure-password = ["bcrypt"]
+# secure-password = ["bcrypt"]
```

Comment:
When I run with this line: secure-password = ["bcrypt"] 
I got an error
"
error: failed to parse manifest at `/Users/wrb/github/wuranbo/gist/rust/workspace/hello_world/Cargo.toml`

Caused by:
  Feature `secure-password` includes `bcrypt` which is neither a dependency nor another feature
"
I did not know how to fix it, so comment. I read the book first time, just a newbie~~ 
And when I saw this line first time, I got nothing context with it on the whole chapter. 

So, tell me, how should I fix it?